packages/bitcoin/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE=https://bitcoincore.org/ 
+TERMUX_PKG_DESCRIPTION="Bitcoin Core" 
+TERMUX_PKG_LICENSE="MIT" 
+TERMUX_PKG_VERSION=0.18.1 
+TERMUX_PKG_SRCURL=https://github.com/bitcoin/bitcoin/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=db3c76ac913adfd67e3c7ff243b443c9706f81dd382d1212875fefc2de1ea5ff 
+TERMUX_PKG_DEPENDS="libevent,libzmq,boost,termux-services"
+TERMUX_PKG_BUILD_DEPENDS="pkg-config,util-linux"
+TERMUX_PKG_BUILD_IN_SRC=1
+TERMUX_PKG_BUILDDIR=$TERMUX_PKG_SRCDIR
```

Comment:
Do not specify `TERMUX_PKG_BUILDDIR=$TERMUX_PKG_SRCDIR`. manually. `TERMUX_PKG_BUILD_IN_SRC` is enough.

Reply:
0001-fixup.patch can't be applied to current source code. This patch is probably outdated.